now.json

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "var": "@var"
+  }
+}
```

Comment:
since we won't be passing env vars and instead we will hardcode the read token, please remove this file